vispy/app/backends/_qt.py

```diff
@@ -266,7 +279,9 @@ def _vispy_get_backend_name(self):
 
     def _vispy_process_events(self):
         app = self._vispy_get_native_app()
-        app.flush()
+        # sendPostedEvents replaces flush which has been removed from Qt6.0+
+        # This should be compatible with Qt4.x and Qt5.x
+        app.sendPostedEvents()
```

Comment:
The Qt4 docs are a little unclear about if flush is the exact same thing as sendPostedEvents (it seems like sendPostedEvents *only* handles events that were explicitly added with postEvent), but seeing as flush is completely deprecated in later APIs I approve of this. Thanks for catching this.